quinn-proto/Cargo.toml

```diff
@@ -36,7 +36,7 @@ slab = "0.4"
 thiserror = "1.0.21"
 tinyvec = { version = "1.1", features = ["alloc"] }
 tracing = "0.1.10"
-webpki = { version = "0.22", default-features = false, optional = true }
+webpki = { version = "0.100.1", package = "rustls-webpki", default-features = false, optional = true }
```

Comment:
Oops... how do we use the direct `webpki` dependency, anyway? Do we still need it? Is releasing this as 0.10.2 semver-compatible?

Reply:
Good questions :sweat_smile:  I spotted it in `cargo tree` looking at the trust-dns update and didn't dig deeper naively assuming `cargo` would warn on an unused explicit dep. 

Looking closer now I don't think it's used anywhere so I've pushed a reworked version of this branch that drops the explicit dependency. I think that will be semver compatible :crossed_fingers: